src/components/common/view/UserAvatar.js

```diff
@@ -35,8 +35,11 @@ export type AvatarProps = {
  */
 const UserAvatar = (props: AvatarProps) => {
   const { profile, children, styles, containerStyle } = props
+  let avatarSize = props.avatarSize
 
-  const avatarSize = getDesignRelativeWidth(AVATAR_DESIGN_WIDTH)
+  if (!avatarSize) {
```

Comment:
1. use .defaultProps
2. then you'll have an 'clean' body with jsx only so you can just return jsx from arrow function

```
const UserAvatar = ({ profile, children, styles, containerStyle, avatarSize }) => (
  <View style={styles.avatar}>
  // ...
)

UserAvatar.defaultProps = {
  avatarSize: getDesignRelativeWidth(AVATAR_DESIGN_WIDTH)
}
```

3. there aren't another 'sizes' and component has 'Avatar' suppix so it's already clear that we passing size of the avatar.
You could rename this prop to just 'size'

Reply:
done